cala-nodejs/src/journal/mod.rs

```diff
@@ -7,7 +7,7 @@ use values::*;
 pub struct NewJournal {
   pub id: Option<String>,
   pub name: String,
-  pub code: String,
+  pub code: Option<String>,
```

Comment:
to match the type for `JournalValues.code`: https://docs.rs/cala-ledger/latest/cala_ledger/balance/struct.JournalValues.html#structfield.code